### ISSUE TITLE
Number of species in the extinct tree

### ISSUE THREAD

**Issue by hyanwong** [OneZoom/tree-build#101]
When we report "number of species", it's clearly wrong in the extinct tree, because a tip like "snakes" only counts as one species. Additionally, there are an indefinite number of species that *could* be in the extinct tree. Here are a few options for fixing the inaccuracy:

* Completely remove the text that reports the number of species (easiest fix)
* Rephrase to say "number of onezoom tips / taxa" (seems ugly)
* Change the calculation code. I think all of these would require a new DB field, however.
    * don't count extant species, and say "number of extinct species/taxa in tree" (this would be relatively easy, I think, although we might fall foul of the times when we display just the genus, and don't expand the leaf to all the species.
    * don't count extinct species but count the number of species in each extant "fake leaf" somehow, and write "number of extant species" (we would need to either get the numbers from another onezoom instance, or use a huge newick + DB and simply not expand the tips). We would also need to distinguish extinct from extant species (probably easiest to do via the extinction_date field).
    * count all species, and do some careful wording to mean "all extant species and all the fossil species we choose to show".

**Comment by davidebbo** [comment]
Yes, this is a problem. I'd rather not remove the text altogether, as I find it useful, if only to know how big my entire tree is (of course, I can get that from the DB). Also, it's mostly a problem in mammals, which mix extinct/extant much more. For Dinosaurs, it's only one node out of (currently) 713, so it's negligible.

Though yes, we should aim to fix it at some point. I think if we can only count extinct tips, it's good enough. Extant tips are only there for giving context.

**Comment by hyanwong** [comment]
For the moment we can phrase round it for the extinct tree, e.g. "number of tips" rather than "number of species"?

**Comment by davidebbo** [comment]
> For the moment we can phrase round it for the extinct tree, e.g. "number of tips" rather than "number of species"?

Yes, I think that's the simplest solution. Maybe we should create an extinct branch in OZTree to make this type of changes (with no intention to merge them, unless we can make the behavior config driven, but let's not worry about that for now).
